edgePy/DGEList.py

```diff
@@ -292,17 +292,40 @@ def rpkm(
         # self = self.cpm(log=log, prior_count=prior_count)
 
     def tpm(
-        self, transcripts: Mapping, log: bool = False, prior_count: float = PRIOR_COUNT
-    ) -> None:
-        """Return the DGEList normalized to reads per kilobase of transcript
-        length.
+        self,
+        gene_lengths: np.ndarray,
+        log: bool = False,
+        prior_count: float = PRIOR_COUNT,
+        mean_fragment_lengths: np.ndarray = None,
+    ) -> np.ndarray:
+        """Normalize the DGEList to transcripts per million
+
+        Args:
+            gene_lengths: vector of length for each gene in counts table
+            log: store log outputs
```

Comment:
I strongly prefer qualifying this variable name with verb or preposition:

So how about `to_log_transform` over `log` which indicates the option is a boolean choice and that the action is an _in situ_ transformation.

https://github.com/r-bioinformatics/edgePy/issues/23#issuecomment-404251095

Reply:
Could you change elsewhere in the codebase so we are consistent?

Reply:
transform_to_log makes it clearer that it's a boolean IMO. Do you think it's appropriate?